lsp/tombi.lua

```diff
@@ -0,0 +1,11 @@
+---@brief
+---
+--- https://tombi-toml.github.io/tombi/
+---
+--- Language server for Tombi, a TOML toolkit.
+---
+return {
+  cmd = { 'tombi', 'lsp' },
+  filetypes = { 'toml' },
+  root_markers = { 'tombi.toml', '.git' },
```

Comment:
I think you want to add `pyproject.toml` to that list as well. See https://tombi-toml.github.io/tombi/docs/configuration.

```suggestion
  root_markers = { 'tombi.toml', 'pyproject.toml', '.git' },
```

Reply:
ddcfe0643f0a14f0accf93f0385f18047a86bd8d